platform/kube/inject/testdata/hello-config-map-name.yaml.injected

```diff
@@ -61,6 +61,7 @@ spec:
         name: istio-config
       - emptyDir:
           medium: Memory
+          sizeLimit: "0"
```

Comment:
Some temporary/hacky solutions that come to mind:

1. custom YAML/JSON marshaler
2. textual removal post-encoding
3. programmatic removal
   * convert typed resource to generic `map[string]interface{}` (https://github.com/fatih/structs maybe?)
   * remove offending key(s)
   * marshal generic map to YAML